brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/IncludeAttachment.kt

```diff
@@ -50,4 +53,17 @@ class AttachmentTemplate<out Lang : LanguageSupport, AttachmentData : VedleggDat
     override fun hashCode() = Objects.hash(title, outline, includeSakspart)
     override fun toString() = "AttachmentTemplate(title=$title, outline=$outline, includeSakspart=$includeSakspart)"
 
+}
+
+class AlltidValgbartVedlegg<out Lang : LanguageSupport>(
```

Comment:
Burde konstruktør være internal?

Reply:
Trur ikkje det, for vi new-ar opp denne i modulane kor han brukast (sjå `AlderTemplates`).